tests/integration/test_data_load_and_clean.py

```diff
@@ -1,22 +1,36 @@
 from document_to_podcast.preprocessing.data_cleaners import (
-    clean_html,
-    clean_with_regex,
     clean_markdown,
+    markdown_to_text,
 )
-from document_to_podcast.preprocessing.data_loaders import load_pdf, load_txt
+from document_to_podcast.preprocessing.data_loaders import load_txt, load_file
+from unittest.mock import Mock
```

Comment:
We are using the `pytest` library for testing, so better to import the Mock object from there

Reply:
Replaced mocker with `from pytest_mock import MockerFixture`.
```
def test_load_and_clean_pdf(example_data, mocker: MockerFixture):
    mock_file = mocker.Mock()
```